Formula/c/cowsay-org.rb

```diff
@@ -0,0 +1,20 @@
+class CowsayOrg < Formula
+  desc "Apjanke's fork of the classic cowsay project"
+  homepage "https://cowsay.diamonds"
+  url "https://github.com/cowsay-org/cowsay/archive/v3.8.3.tar.gz"
+  sha256 "3bcb1f644a85792bc2ee8601971f16f8f1e7ca0013d6062cf35b4fd6d8fa29ea"
+  head "https://github.com/cowsay-org/cowsay.git", branch: "main"
+  license "GPL-3.0-only"
+
+  conflicts_with "cowsay", :because => "cowsay-org is a fork of cowsay"
```

Comment:
since cowsay is already disabled, maybe just update that formula to use the fork directly?